samples/drivers/ipm/ipm_esp32/CMakeLists.txt

```diff
@@ -4,12 +4,12 @@ cmake_minimum_required(VERSION 3.20.0)
 
 set(REMOTE_ZEPHYR_DIR ${CMAKE_CURRENT_BINARY_DIR}/ipm_esp32_appcpu-prefix/src/ipm_esp32_appcpu-build/zephyr)
 
-if("${BOARD}" STREQUAL "esp32_devkitc_wrover")
-  set(BOARD_REMOTE "esp32_devkitc_wrover_appcpu")
-elseif("${BOARD}" STREQUAL "esp32_devkitc_wroom")
-  set(BOARD_REMOTE "esp32_devkitc_wroom_appcpu")
-elseif("${BOARD}" STREQUAL "esp32s3_devkitm")
-  set(BOARD_REMOTE "esp32s3_devkitm_appcpu")
+if("${BOARD}" STREQUAL "esp32_devkitc_wrover/esp32/procpu")
```

Comment:
No change requested and I know it will be addressed later.

In general this kind of testing is dangerous because typos or renames are not caught by compliance, whereas a test for `CONFIG_BOARD_<full_identifier>` where the config has a typo / rename would fail compliance.

Secondly the split in board + identifier, where clusters are inherited, means that sysbuild will be able to check for just the CPU cluster, and adjust only the cluster identifier, and leave the rest, board + soc + variant identical, thus any board using esp32 or esp32s3 can be supported with a simple test.